mobile/src/main/res/layout/card_view_price_dialog.xml

```diff
@@ -17,104 +17,256 @@
   ~ along with MTG Familiar.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
I don't know how you feel about it, but it was suggested to me to add a toggle to view normal vs foil prices. I'm not sure how I feel about that either. What I thought was a table of prices. It would be something to mock up and compare to see what works best.

Reply:
It's already a table? I'm against the toggle (any user interaction, really) when there's enough screen space to display everything.